### DISCUSSION TITLE
multiple item carousel

### DISCUSSION THREAD

**Discussion by subrayamallya** [twbs/bootstrap#33425] [Ideas]
Hi
I searched to make sure this is not already reported/requested by someone. Has the community/team discussed implementing a carousel that supports multiple items like this one https://www.espncricinfo.com

currently it requires a bunch of JS, CSS changes and custom code to make it do as explained in this https://azmind.com/demo/bootstrap-carousel-multiple-items/. Seems like something Bootstrap should support out of the box with a multiple="yes" (in combination with maybe grid)

**Comment by ascensus-mdb** [comment]
Here you can find an example for v5. No additional CSS or JS.

https://mdbootstrap.com/docs/standard/extended/gallery/#section-multi-item-carousel

It uses MDBootstrap, but actually, it's not necessary to make it work. You can easily implement this solution in regular Bootstrap as well

**Reply by subrayamallya** [reply]
Thank you 🙏🏻
